### PR TITLE
update documentation

### PR DESCRIPTION
**Pull request by rosesyrett**
This repository has changed a bit and the docs need to be updated.

### CODE REVIEW

**Comment by RAYemelyanova** [comment]
Will also want to update lifecycle.rst

File: docs/developer/explanations/architecture.rst [hunk 1]
```diff
@@ -37,10 +37,17 @@ the known expectations of the plan, passes it to the ``RunEngine`` and handles a
 The Service Object
```
Comment (keithralphs) [inline docs/developer/explanations/architecture.rst:37]: It's now called handler but maybe should have a better name

File: docs/developer/how-to/update-openapi.rst [hunk 2]
```diff
@@ -0,0 +1,14 @@
+Update the openapi schema
+-------------------------
+
+If you change any of the code in src/blueapi/service/main, it is imperative that
+you update the openapi schema to reflect these changes. There is a test to check
```
Comment (keithralphs) [inline docs/developer/how-to/update-openapi.rst:5]: add reason why

File: docs/images/blueapi-architecture.png
Comment (keithralphs) [inline docs/images/blueapi-architecture.png:1]: check how this shows up in dark mode

File: docs/user/how-to/use-cli.rst [hunk 3]
```diff
@@ -0,0 +1,39 @@
+Use the command line interface (CLI)
+====================================
+
+Blueapi comes pacakged with a simple click based CLI. You can start the server
```
Comment (keithralphs) [inline docs/user/how-to/use-cli.rst:4]: delete first sentence

File: docs/user/how-to/use-cli.rst [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+and then query it for plans and devices, as well as ask it to run a plan, through
+the CLI.
```
Comment (keithralphs) [inline docs/user/how-to/use-cli.rst:6]:
```suggestion
the bundled CLI client.
```

File: docs/user/how-to/use-cli.rst [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+Starting the server
```
Comment (keithralphs) [inline docs/user/how-to/use-cli.rst:8]: This needs merging with the out of date cli docs in main (https://diamondlightsource.github.io/blueapi/main/user/how-to/run-cli.html) an requires a link to https://diamondlightsource.github.io/blueapi/main/user/reference/cli.html

**Comment by stan-dot** [comment]
this is by definition out of date, petition to close @callumforrester

**Comment by callumforrester** [comment]
@stan-dot would you mind reviewing it and making sure there is no useful information that's still relevant? If so you can close it.

**Comment by stan-dot** [comment]
no, there's no useful information at this point

**Pull request closed without merging**